assets/js/components/ChecksResults/ChecksResults.jsx

```diff
@@ -29,6 +29,7 @@ import {
   sortHosts,
   sortChecks,
 } from './checksUtils';
+import ClusterInfoBox from '../ClusterDetails/ClusterInfoBox';
```

Comment:
I would go with `@components/Clust...` here.
As it belongs to other component, let's use the alias. Otherwise, we can have issues if we change folders or anything

Reply:
Xabi's right, also, I would turn this to `import { ClusterInfoBox } from '@components/ClusterDetails` adjusting `ClusterDetails`' `index.js` file accordingly